frontend/src/workbench/Workbench.types.ts

```diff
@@ -16,6 +16,7 @@ export type Selection = {
   id: string;
   label: string;
   kind: string;
+  descriptionId?: string;
```

Comment:
As discussed this morning too, this should wait for the contribution related to the ADR-36 in order to leverage some additional information on the kind of a graphical selection for example.